view/adminhtml/templates/new_shipment.phtml

```diff
@@ -42,6 +42,7 @@
                value="1"
                type="checkbox"
                 <?= $block->getDefaultOption('only_recipient') == true ? 'checked="checked"' : '';?>
+                <?= $block->getDefaultOption('age_check') == true ? 'checked="checked"' : '';?>
```

Comment:
Je kan hier 2x een checked hebben:
dan krijg je:
checked="checked" checked="checked"

Reply:
Dat is niet op de pagina te zien.